README.md

```diff
@@ -1,57 +1,9 @@
-# Northcoders News API
+files to add to connect to the two databases:
+.env. files
```

Comment:
I'd explain they need two .env files. One for test one for dev - explain how they need to look